documentation/docs/getting_started/nodes_101.md

```diff
@@ -52,14 +52,14 @@ All nodes keep a record of the [Unspent Transaction Outputs (UTXO)](https://chry
 
 * Check that a transaction is not transferring more IOTA tokens than are available on the address.
```

Comment:
the link on line 51 should point to the [wiki](https://wiki.iota.org/chrysalis-docs/guides/developer#unspent-transaction-output-utxo)